nengo_viz/static/viz_sim_control.js

```diff
@@ -37,15 +37,19 @@ VIZ.SimControl = function(div, args) {
     
     /** Create the pause button */
     this.pause_button = document.createElement('button');
-    this.pause_button.innerHTML='Pause';
+    this.pause_button.className = "btn btn-default play-pause-button";
+    this.pause_button.innerHTML='<i class="glyphicon glyphicon-pause"></i>';
```

Comment:
The <i> tag is kind of an industry standard for using icons (used by facebook and twitter). The alternative would be to use <span> tags. I don't really have a preference of which way to do it, but here's a good comparison can be found here: http://stackoverflow.com/questions/11135261/should-i-use-i-tag-for-icons-instead-of-span